server/src/remote.rs

```diff
@@ -545,6 +548,55 @@ impl FlightService for FlightServiceHandler {
 
             // Confirm the data was flushed.
             Ok(Response::new(Box::pin(stream::empty())))
+        } else if action.r#type == "CollectMetrics" {
```

Comment:
Sorry I did not think about it when performing the initial review, but maybe update the doc comment for `do_action()` as it currently states that `CommandStatementUpdate` is the only supported `Action`. The doc comment also currently contains a lot of detail about `CommandStatementUpdate` so maybe this should be reduced if information about the other `Action`s are added.

Reply:
Good catch, fixed.